Makefile

```diff
@@ -1,10 +1,17 @@
-.PHONY: depend up down
+.PHONY: depend set-region up down
 
 depend:
 	ansible-galaxy install -r ./provision/requirements.yml
 
-up: depend
+set-region:
+	export AWS_DEFAULT_REGION=us-west-2
+
+up:
```

Comment:
`depend` and `set-region` can be specified as dependencies in this line instead of explicitly calling make commands

Reply:
so just to clarify - if writing it as follows:
`up: depend set-region
	ansible-playbook --tags up ./provision/main.yml
`
Does the makes the command run the 2 commands before it runs the ansible-playbook? @lmilbaum